modules/insights-operator.adoc

```diff
@@ -17,12 +17,10 @@ ifdef::operator-ref[= Insights Operator]
 ifdef::cluster-caps[= Insights capability]
 
 ifdef::operator-ref[]
-
 [NOTE]
 ====
 The Insights Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Installing_.
```

Comment:
```suggestion
The Insights Operator is an optional cluster capability that cluster administrators can disable during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Installing_.
```
Consider adding a link to the "Cluster capabilities" section in the "Additional resources" subsection.